doc/install.text

```diff
@@ -0,0 +1,124 @@
+created as well:
+
+echo $DRP_STELLA_DIR
+
+should show you:
+
+~/stella-git/drp_stella
+
+
+* Build the pipeline:
+
+cd drp_stella
+scons opt=3 -j8
+
+
+* Create the defects database (done during compilation of the code):
+
+cd ../obs_pfs
+scons opt=3 -j8
+
+
+* Now for using the pipeline. Raw test data are in $DRP_STELLA_DATA_DIR/tests/data/raw/ (3 Biases, 3 Darks, 1 Flat, 1 Arc). 
+First we need to create a directory (actually 2) where we want to store pipeline outputs. 
+Let's assume you have a directory ~/spectra and do everything related to spectra there:
+
+mkdir ~/spectra/PFS
+mkdir ~/spectra/PFS/CALIB
+
+
+* Configuration parameters can be passed to the pipeline task either on the command line or in a config file 
+(e.g. $OBS_PFS_DIR/config/pfs/ingest.py, see following example). You can list all possible configuration parameters by 
+appending a "--show config" to the parameter list.
+To ingest the raw images:
+
+cd ~/spectra
+ingestImages.py PFS $DRP_STELLA_DATA_DIR/tests/data/raw/*.fits --output=PFS --mode=link -C $OBS_PFS_DIR/config/pfs/ingest.py
+
+
+* You can now inspect the created SQL database with e.g. sqlitebrowser (http://sqlitebrowser.org/).
+Now that we have our database we can start reducing things. We should probably start with the Biases. If the biases we
+want to reduce were observed on 2015-12-22 on spectrograph 2, arm red (r) at site S:
+
+reduceBias.py PFS --calib PFS/CALIB --output PFS --calibId calibVersion=bias arm=r calibDate=2015-12-22 spectrograph=2 --do-exec --id field=BIAS dateObs=2015-12-22 --clobber-config --nodes=1 --procs=1
+
+
+* Note the 2 config parameters --nodes and --procs at the end. These parameters are required by tasks which are 
+parallelized. Sometimes running the code in parallel can lead to problems (in most cases caused by the 3rd-party libraries
+used), so setting nodes and procs to 1 is a safe choice. —-clobber-config is needed if you re-run the task with different
+config parameters. The config used for each run is written to PFS/config/bias.py and the task won’t run again if you 
+change parameters, unless you say —-clobber-config.
+
+* Now that we have a master bias we need to inject that too into our database:
+
+genCalibRegistry.py --root=PFS/CALIB --camera PFS --validity 180
+
+* Now we can create a bias-subtracted Dark and ingest that into our database:
+
+reduceDark.py PFS --calib PFS/CALIB --output PFS --calibId calibVersion=dark arm=r calibDate=2015-12-22 spectrograph=2 --do-exec --id field=DARK dateObs=2015-12-22 --nodes=1 --procs=1 -C ~/stella-git/obs_pfs/config/pfs/dark.py
+genCalibRegistry.py --root=PFS/CALIB --camera PFS --validity 180
+
+* and a bias and dark-subtracted flat (currently only used to trace the apertures of the fiber traces). Note that in this
+example we set the config parameters isr.doBias and isr.doDark to True after <—-config > (instead of the parameter file
+after <-C > in the ingestImages.py example above):
```

Comment:
This relates to [my comment on the ticket](https://pfspipe.ipmu.jp/jira/browse/INFRA-36?focusedCommentId=11225) — what's your audience for this guide? I'd suggest that what you want to provide is basic information about running drp_stella, not how to use the pipe_base framework, but perhaps that's something we could discuss if you disagree.